npm/private/npm_translate_lock_state.bzl

```diff
@@ -426,13 +426,26 @@ def _copy_input_file(priv, rctx, label_store, key):
         fail("key not found '{}'".format(key))
 
     if _should_update_pnpm_lock(priv):
-        # NB: rctx.read will convert binary files to text but that is acceptable for
-        # the purposes of calculating a hash of the file
-        _set_input_hash(
-            priv,
-            label_store.relative_path(key),
-            utils.hash(rctx.read(label_store.path(key))),
-        )
+        path = label_store.path(key)
+        written = False
+        if path.endswith("package.json"):
+            content = json.decode(rctx.read(path))
```

Comment:
Do we care about handling failures here? I guess we parse package.json in the exact same way elsewhere so it doesn't matter?